gmail/style.css

```diff
@@ -980,10 +980,27 @@ html.simpl .aSs {
 	background-color: rgba(0,0,0,.7);
 }
 
+
+/* Simplify formatting options. Remove shadow and make full width. */
+html.simpl .aoI .Ur {
+	width: 100%;
+	margin: auto;
+	box-sizing: border-box;
+	background-color: white;
+}
+
+html.simpl .aoI .Ur div[aria-label='Formatting options'] {
```

Comment:
This depends on the interface being in English. `html.simpl .aoI .Ur div[role="toolbar"]` should work across languages as this is not a user facing string.

Reply:
Common sense :) . I will make the changes.

Reply:
Updated.